app/assets/javascripts/notifications.js

```diff
@@ -141,9 +141,12 @@ var onReadyNotifications = function() {
   });
 
   $('.tip_notifications_text').click(function(event) {
-    event.stopPropagation();
+    if(! $(event.target).hasClass("kick")){
```

Comment:
Just for stylistic purposes let's do `if (!$(event.target).hasClass('kick')){`
Also just for future reference, let's add a brief comment above describing why this was done

Reply:
Can you update this as I requested? ^